### DISCUSSION TITLE
File/s interface drag and drop file

### DISCUSSION THREAD

**Discussion by Nitwel** [directus/directus#4794] [Feature Requests]
It would be nice to drag and drop a file directly onto any file related interface.

File interface:
If no file exists upload else override the file.
Files interface
If no file exists upload
If files exist either drag onto file to override or drag to special section to upload

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
